src/services/ios/afc/index.ts

```diff
@@ -0,0 +1,473 @@
+import { logger } from '@appium/support';
+import { readFile } from 'node:fs/promises';
+import net from 'node:net';
+import path from 'node:path';
+
+import {
+  buildClosePayload,
+  buildFopenPayload,
+  buildReadPayload,
+  buildRemovePayload,
+  buildRenamePayload,
+  buildStatPayload,
+  nextReadChunkSize,
+  parseCStringArray,
+  parseKeyValueNullList,
+  readAfcResponse,
+  rsdHandshakeForRawService,
+  sendAfcPacket,
+  writeUInt64LE,
+} from './codec.js';
+import {
+  AFC_FOPEN_TEXTUAL_MODES,
+  AFC_RSD_SERVICE_NAME,
+  AFC_WRITE_THIS_LENGTH,
+  MAXIMUM_READ_SIZE,
+} from './constants.js';
+import { AfcError, AfcOpcode } from './enums.js';
+
+const log = logger.getLogger('AfcService');
+
+export interface StatInfo {
+  st_ifmt: string;
+  st_size: number;
+  st_blocks: number;
+  st_mtime: Date;
+  st_birthtime: Date;
+  st_nlink: number;
+  LinkTarget?: string;
+  [k: string]: any;
+}
+
+/**
+ * AFC client over RSD (Remote XPC shim).
+ * After RSDCheckin, speaks raw AFC protocol on the same socket.
+ */
+export class AfcService {
+  static readonly RSD_SERVICE_NAME = AFC_RSD_SERVICE_NAME;
```

Comment:
```suggestion
  static readonly RSD_SERVICE_NAME = 'com.apple.afc.shim.remote';
```

No need to use the exported const IMO. We do the same for rest of the services.

Reply:
The rest of the services have a single index file, so there wasn’t really a need to use constants in the first place.
AFC, on the other hand, has quite a few constants.

so thought to move to constants within afc !
i will move if its confusing.